File: lista3casa/10.py
while True:
    op = input("Digite uma operação (+, -, *, /) ou 'S' para sair: ")
    if op=='S':
        print("Saindo do programa...")
        break
    if op in ['+', '-', '*', '/']:
        a = float(input("Digite o primeiro número: "))
        b = float(input("Digite o segundo número: "))
        
        if op=='+':
            r = a + b
            print("Resultado:", r)
        elif op=='-':
            r = a - b
            print("Resultado:", r)
        elif op== '*':
            r = a * b
            print("Resultado:", r)
        elif op== '/':
            if b != 0:
                r = a / b
                print("Resultado:", r)
            else:
                print("Erro: Divisão por zero não é permitida.")
    else:
        print("Operação inválida. Tente novamente.")

File: listas/lista3casa/10.c
#include <stdio.h>
#include <stdlib.h>
int main(){
float a,b,r;
char op;
while(1){
printf("digite uma operação(+, -, *, /) ou 's' para sair: ");
scanf("%c", &op);
if (op=='s'){
        printf("saindo do programa...");
        break;
    }
if (op== '+' || op == '-' || op == '*' || op == '/'){
    printf("digite o primeiro numero:");
    scanf("%f",&a);
    printf("digite o segundo numero:");
    scanf("%f",&b);
    if(op=='+'){
        r=a+b;
        printf("resultado=%.2f", r);
    }
    else if(op=='-'){
        r=a-b;
        printf("resultado=%.2f", r);
    }
    else if(op=='*'){
        r=a*b;
        printf("resultado=%.2f", r);
    }
    else if(op=='/'){
        if (b!=0){
            r=a/b;
            printf("resultado=%.2f", r);
        }
        else{
            printf("erro: divisão por zero não é permitida.");
        }
    }
}
else{
    printf("operação invalida, tente novamente.");
}
}
system("pause");
return 0;
}